packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2679,16 +2679,37 @@ curl -v -X POST \
 -H "Accept: application/json" \
 -H "Content-Type: application/json" \
 -H "Authorization: SSWS ${api_token}" \
-"https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/reset_password?provider=FEDERATION&sendEmail=false"
+"https://${yourOktaDomain}/api/v1/users/{userId}/lifecycle/reset_password?provider=FEDERATION&sendEmail=false"
 ```
 
 ##### Response example
 
-
 ```json
 {}
 ```
 
+##### Request example (Convert a user to an Okta User)
```

Comment:
"an user"?

Reply:
No, "a user" is correct in this scenario. English is a funny language.